tests/test_iconify.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from qtpy.QtWidgets import QPushButton
+
+from superqt import QIconifyIcon
+
+
+def test_qiconify(qtbot):
+    pytest.importorskip("pyconify")
+    icon = QIconifyIcon("bi:alarm-fill", color="red", rotate=90)
```

Comment:
```suggestion
def test_qiconify(qtbot, tmp_path):
    pytest.importorskip("pyconify")
    icon = QIconifyIcon("bi:alarm-fill", color="red", rotate=90, dir=str(tmp-path))
```

keep test artifacts in test dirs

Reply:
Actually we’ll need to mock the pyconify cache here too